packages/expo-background-task/ios/BackgroundTaskTester.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This interface/implementation in Objective-C ensures that the triggerBackgroundTaskTest is
+ * only compiled in debug mode. There is some uncertainty about whether Swift's #if
+ * conditionals work the same way as in Objective-C.
```

Comment:
yes, swift supports `#if DEBUG`. i had the same question before and found a proof.

Reply:
Awesome - good to know. I've implemented it in Objective-c for now :)